source/uwp/Renderer/lib/Util.cpp

```diff
@@ -547,6 +547,15 @@ HRESULT GenerateSharedSeparator(
 
 HRESULT GetColorFromString(std::string colorString, ABI::Windows::UI::Color *color) noexcept try
 {
+    // Expected format for color string: "#AARRGGBB"
+    // Other format is ignored (set alpha to 0)
+    if (colorString.length() != 9)
+    {
+        color->A = static_cast<BYTE>(0);
+
+        return S_OK;
+    }
```

Comment:
this do validate that the color complies with the format #AARRGGBB, but how are we validating that the values for ARGB are valid hex characters??

Reply:
you're right! Fixed in 0b983df